### ISSUE TITLE
Maintainance?

### ISSUE THREAD

**Issue by whimsicaldreamer** [jquery-form/form#508]
Is this plugin no longer actively maintained? If no then why is there no new updates to this plugin?

**Comment by alienintheheights** [comment]
It hasn't been updated in three years. It looks very dead. Last release has issues with IE Edge. We're in the process of removing it from our products.

**Comment by federicovilla** [comment]
Would be great to see this library updated and maintained...it's one of the best around here

**Comment by alienintheheights** [comment]
agree! Migrating is a pain. We're looking at Fine Uploader, btw.

**Comment by kevindb** [comment]
I have forked the repo to an organization so that we can maintain the plugin as a community. I'll try to do some work cleaning up and examining pull requests this weekend.
https://github.com/jquery-form/form

**Comment by kevindb** [comment]
This issue was moved to jquery-form/form#1

**Comment by kevindb** [comment]
TLDR: Yes jQuery Form plugin is being maintained.

The original owner transferred ownership of the repo to the jquery-form Github organization today so the project won't be dependent on one maintainer anymore. I'm working on accepting pull requests and getting the project back into shape.
